scripts/upgradeRabbitHoleTickets.js

```diff
@@ -0,0 +1,28 @@
+require('dotenv').config()
+require('@nomiclabs/hardhat-ethers')
+const { ethers } = require('hardhat')
+
+async function main() {
+  hre.run("compile");
+  const address = "0xC5340D9c3bDE433492D1Af95AD353B74C7DbD98C"; // goerli
```

Comment:
How are we going to handle different envs here? (In prev scripts I just listed each network and commented what I wasn't using lol)

Reply:
Probably just do that (comment, un-comment) for now.